.pylintrc

```diff
@@ -9,27 +9,49 @@ init-hook='import sys; sys.path.append("../oppia_tools/google_appengine_1.9.67/g
 
 # Checks for correct docstring style
 load-plugins=pylint.extensions.docstyle
+ignore=.vscode
+
+# Use multiple processes to speed up Pylint.
+jobs=1
 
 [BASIC]
 
-# Regular expression which should only match correct function names
-function-rgx=^[_a-z][a-z0-9_]*$
+# Naming style matching correct argument names
+argument-naming-style=snake_case
 
-# Regular expression which should only match correct method names
-method-rgx=^([_a-z][a-z0-9_]*|__[a-z0-9]+__)$
+# Naming style matching correct attribute names
+attr-naming-style=snake_case
 
 # Regular expression which should only match correct constant names
 const-rgx=^(([A-Z_][A-Z0-9_]*)|(__.*__)|([a-z_]+_models)|([a-z_]+_services))$
 
-# Good variable names which should always be accepted, separated by a comma
-good-names=e,_,d,f,i,l,p,w,fn,fs,id,pc,sc,zf,setUp,tearDown,longMessage,maxDiff
+# Naming style matching correct constant names
+const-naming-style=UPPER_CASE
+
+# Naming style matching correct class names
+class-naming-style=PascalCase
 
 # Regex for dummy variables (to prevent 'unused argument' errors)
 dummy-variables-rgx=_|unused_*
 
+# Regular expression which should only match correct function names
+function-rgx=^[_a-z][a-z0-9_]*$
+
+# Good variable names which should always be accepted, separated by a comma
+good-names=e,_,d,f,i,l,p,w,fn,fs,id,pc,sc,zf,setUp,tearDown,longMessage,maxDiff
```

Comment:
Not sure you need most of this FYI.

Reply:
Got rid of most of them except `setUp` and `tearDown`!